syncto/views/collection.py

```diff
@@ -0,0 +1,68 @@
+from pyramid import httpexceptions
+from pyramid.security import NO_PERMISSION_REQUIRED, forget
+
+from cliquet import Service
+from cliquet.errors import http_error, ERRORS
+from sync.client import SyncClient
+
+from syncto.utils import base64_to_uuid4
+
+
+collection = Service(name='collection',
+                     description='Get the Firefox Sync Collection',
+                     path='/{collection_name}',
+                     cors_headers=('Next-Page', 'Total-Records',
+                                   'Last-Modified', 'ETag'))
+
+AUTHORIZATION_HEADER = 'Authorization'
+CLIENT_STATE_HEADER = 'X-Client-State'
+
+
+@collection.get(permission=NO_PERMISSION_REQUIRED)
+def collection_get(request):
+    collection_name = request.matchdict['collection_name']
+    # Get the BID assertion
+    if AUTHORIZATION_HEADER not in request.headers or \
```

Comment:
Actually it was during a peer review with @n1k0 where I was writting comments @n1k0 and I talked about.